firmware/application/apps/ui_looking_glass_app.cpp

```diff
@@ -184,14 +210,22 @@ void GlassView::on_channel_spectrum(const ChannelSpectrum& spectrum) {
             }
         }
         // process actual bin
-        if (process_bins(&max_power) == true)
+        if (process_bins(&max_power)) {
+            uint8_t real_beep_squelch = beep_squelch * 255 / 100;
+
+            if (range_max_power >= real_beep_squelch) {
+                baseband::request_audio_beep(400 + (((256 - range_max_power) * 2600) / 255), 24000, 500);
```

Comment:
On line 58 you've programmed the sample clock for 48KHz but here you're telling baseband that you've programmed it for 24KHz.  This will result in the actual audio beep frequency being double what you're requesting on this line.

Reply:
Fixed, thanks